### DISCUSSION TITLE
Light loses all settings data randomly and after power cycle

### DISCUSSION THREAD

**Discussion by nickdos** [arendst/Tasmota#17235] [Q&A] [answered]
I have 2 identical light bulbs (Tasmota 12.2.0) with the same template settings and one of them randomly resets and loses its template settings (often after a power cycle but not always). The only fix is to restore settings via backup file. I've checked that it has these `status 0` values:

`"SaveData":1,"SaveState":1`

I'm guessing that its probably a fault with the bulb but just wanted a sanity check in case there is some magic I can perform to restore long term data retention.

**Comment by barbudor** [comment, marked as answer]
If power cycle is not clean, it may trigger the [self reset recovery](https://tasmota.github.io/docs/Device-Recovery/#fast-power-cycle-device-recovery) procedure
As documented you can disable this procedure using [`SetOption65 1`](https://tasmota.github.io/docs/Commands/#fast-power-cycle-device-recovery)

Of course disabling this procedure set away any chance to be able to reset the configuration the day you will need it.

**Reply by nickdos** [reply]
Thanks for the reply. I’m not sure that is the cause but it did make me think about it a bit more and I’ll try swapping the 2 bulbs to see if there is something about the wiring that might be causing a dirty power cycle. If the same bulb keeps resetting, I’ll just replace it.

**Reply by sfromis** [reply]
One risk factor is if your Tasmota setup drives the bulb harder (hotter) than the factory setup, leading to overheating and failures. You could try if it is more stable at a much lower brightness. Tasmota does have features like `DimmerRange` if you need to restrict how hard you push the hardware.

**Reply by nickdos** [reply]
That's a good point. I'm running the 2 bulbs as a "pair", so I'm lucky to have a control. The other bulb is fine being driven at the same brightness so it does suggest a hardware issue. The bulbs are <$20usd so not a huge deal to replace.
